### PR TITLE
php: Update Compression example

### PR DESCRIPTION
**Pull request by jeffreyqw**
Compression Helloworld example and Compression readme

@stanley-cheung

### CODE REVIEW

**Comment by stanley-cheung** [comment]
This is great - but I am thinking we probably don't need a separate `examples/php/Compression` directory just for this. It's quite a bit of copied code from the same `greeter_client.php`. 

So I was thinking we should add a new section under this: https://github.com/grpc/grpc/tree/master/src/php#maximum-message-size

That is where we document what some extra options you can pass to the channel. That will be a good place for this.

Thanks!

**Comment by stanley-cheung** [comment]
Cool, but please remove these two files: examples/php/Compression/CompressionGreeterClient.php and examples/php/Compression/README.md.

Also, this line needs to go into the bottom https://github.com/grpc/grpc/pull/22841/files#diff-6789d16e9e494efbc61989c74f3dd38eL368

File: src/php/README.md [hunk 1]
```diff
@@ -365,4 +365,30 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
 ]);
 ```
 
-[Node]:https://github.com/grpc/grpc/tree/master/src/node/examples
+### Compression 
+
+You can customize the compression behavior on client side, specify this `grpc.default_compression_algorithm (int)` option control compression algorithm for the channel and `grpc.default_compression_level (int)` option control  compression level for the channel.
```
Comment (stanley-cheung) [inline src/php/README.md:370]: "You can customize the compression behavior on the client side, by specifying the following options when constructing your PHP client."

File: src/php/README.md [hunk 2]
```diff
@@ -365,4 +365,30 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
...
+
+``` 
+Possible values for compression algorithm for the channel are:
```
Comment (stanley-cheung) [inline src/php/README.md:373]: "Possible values for `grpc.default_compression_algorithm`:"

File: src/php/README.md [hunk 3]
```diff
@@ -365,4 +365,30 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
...
+0 - No compression
+1 - Compress with DEFLATE algorithm
+2 - Compress with GZIP algorithm
+3 - Stream compression with GZIP algorithm
+```
+```
+Possible values for compression level for the channel are:
```
Comment (stanley-cheung) [inline src/php/README.md:380]: "Possible values for `grpc.default_compression_level`:"

File: src/php/README.md [hunk 4]
```diff
@@ -365,4 +365,30 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
...
+0 - None
+1 - Low level
+2 - Medium level
+3 - High level
+```
+Here's how you can specify compression behavior when constructing your PHP client:
```
Comment (stanley-cheung) [inline src/php/README.md:386]: Here's an example on how you can put them all together:

File: src/php/README.md [hunk 5]
```diff
@@ -365,4 +365,34 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
 ]);
 ```
 
-[Node]:https://github.com/grpc/grpc/tree/master/src/node/examples
+### Compression 
+
+You can customize the compression behavior on the client side, by specifying the following options when constructing your PHP client.
+
+`grpc.default_compression_algorithm (int)` option control compression algorithm for the channel.
```
Comment (stanley-cheung) [inline src/php/README.md:372]: Please remove these two lines

File: src/php/README.md [hunk 6]
```diff
@@ -365,4 +365,34 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
...
 ```
 
-[Node]:https://github.com/grpc/grpc/tree/master/src/node/examples
+### Compression 
+
+You can customize the compression behavior on the client side, by specifying the following options when constructing your PHP client.
+
+`grpc.default_compression_algorithm (int)` option control compression algorithm for the channel. 
+
+`grpc.default_compression_level (int)` option control compression level for the channel.
```
Comment (stanley-cheung) [inline src/php/README.md:374]: This one too

**Comment by stanley-cheung** [comment]
And can you squash these commits down to 1 please? Thanks.

File: src/php/README.md (hunk 6 above)
Comment (jeffreyqw) [reply src/php/README.md]: got it.

**Pull request closed without merging**